lib/elasticsearch_rasi/lang_to_country.rb

```diff
@@ -0,0 +1,30 @@
+# encoding:utf-8
+require_relative 'document'
+class ElasticsearchRasi
```

Comment:
ještě mám dvě otázky:
1/ proč je to modul (uznávám, že ostatní jsou, ale je to blbě :))
2/ jak se to bude volat?

Reply:
1. myslel jsem, že když z toho budu volat jen jednu funkci, bude lepší modul než třída 
2. předpokládal jsem, že  `@es.translate_lang_to_country(language)`

Reply:
Ani moc ne, respektive, modul by se k tomu používat neměl. Spíše udělat statickou metodu a pak nějakou public metodu která bude volat třídu TranslateLangToCountry ... ?

Reply:
Upraveno + přidán test